src/Components/DataHistoryExplorer/DataHistoryExplorer.tsx

```diff
@@ -31,12 +31,17 @@ const getClassNames = classNamesFunction<
 >();
 
 const DataHistoryExplorer: React.FC<IDataHistoryExplorerProps> = (props) => {
```

Comment:
where does this data come from now and why didn't we need it before for teh builder?

Reply:
which data are you referring to? is it the optional timeSeriesTwins? If so, as I explained in my other comment: Because in one of your comments you asked for a story with the data already filled in (which might be also needed to feed in the modal with the selected twin&property in the property inspector anyhow), that is why I added this prop to open the data history explorer modal with a already added time series information. Does it make sense?

Reply:
I was seeing hte twin data showing up as a new prop in the most recent commits. Ideally, we wouldn't add a prop just for the story. My intention in teh story was to have some interactions that would select the right things to populate the list. Let's not pollute the production props with props only for testing, that is likely to confuse us down the road.